intern/remote/cv/remote.py

```diff
@@ -0,0 +1,278 @@
+﻿"""
+# Copyright 2020 The Johns Hopkins University Applied Physics Laboratory
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+from intern.remote import Remote
+from intern.resource.cv.resource import CloudVolumeResource
+from intern.service.cv.volume import VolumeService
+from intern.service.cv.metadata import MetadataService
+from intern.service.cv.project import ProjectService
+
+
+class CloudVolumeRemote(Remote):
+    def __init__(
+        self, cfg_file_or_dict=None,
+    ):
+        """Constructor.
+        Protocol and host specifications are taken in as keys -values of dictionary.
```

Comment:
We should include what arguments we expect to receive here in the docstring

Reply:
Done